pom.xml

```diff
@@ -217,7 +216,7 @@
             <dependency>
                 <groupId>org.elasticsearch</groupId>
                 <artifactId>elasticsearch</artifactId>
-                <version>2.1.1</version>
+                <version>2.4.6</version>
```

Comment:
there seems to be a problem in Sandbox where the reindex is not working

this ES lib upgrade may cause the issue (a `jackson-dataformat-yaml` depedency clash causing a problem where `snakeyaml` 1.15 is now being used (and failing at reindex time) where we used to have `snakeyaml` 2.2 working

I am attempting a quick fix, but this release should not go in until it works with reindex